content/docs/command-reference/version.md

```diff
@@ -107,12 +109,14 @@ Inside a DVC project:
 ```dvc
 $ dvc version
 
-DVC version: 1.7.4 (pip)
+DVC version: 1.10.0 (pip)
```

Comment:
just to not confuse people with the older DVC version

Reply:
No strong opinion but there's another 3 instances of `1.7` in this doc.

Reply:
Updated, thanks!